Purchases/SubscriberAttributes/RCSubscriberAttribute.m

```diff
@@ -0,0 +1,100 @@
+//
+// Created by RevenueCat on 2/17/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RCSubscriberAttribute.h"
+#import "RCDateProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+#define KEY_KEY @"key"
+#define VALUE_KEY @"value"
+#define APP_USER_ID_KEY @"appUserID"
+#define SET_TIME_KEY @"setTime"
+#define IS_SYNCED_KEY @"isSynced"
+
+#define BACKEND_VALUE_KEY @"value"
+#define BACKEND_TIMESTAMP_KEY @"updated_at"
+
+
+@interface RCSubscriberAttribute ()
+
+@property (nonatomic, copy) NSString *key;
+@property (nonatomic, copy) NSString *value;
+@property (nonatomic, copy) NSString *appUserID;
```

Comment:
why are you storing the appUserID? Aren't you going to store in a different `NSUserDefaults` key?

Reply:
I originally thought I'd end up using it, but that wasn't the case. I removed it from this and subsequent PRs. Great catch!